src/Main.java

```diff
@@ -17,11 +17,13 @@ public static void main(String[] args) {
         pokemons.add(pok4);
```

Comment:
"pok4" is een nietszeggende variabele naam. Beter zou zijn om "Serperior" te gebruiken als variabele naam.